helusers/pipeline.py

```diff
@@ -13,6 +13,25 @@
 logger = logging.getLogger(__name__)
 
 
+def ensure_uid_is_uuid(details, backend, response, user=None, *args, **kwargs):
+
+    uid = kwargs.get('uid')
+
+    if is_valid_uuid(uid):
+        return None
+
+    # django-helusers uses UUID as the primary key for the user.
+    # If the incoming token does not have UUID in the sub field,
+    # we must synthesize one
+    if not is_valid_uuid(uid):
```

Comment:
Isn't this check needless, since the inverse check is just above and the function exits in that case?

Reply:
Good point. This is probably the same problem in code that I have with natural language. Ie. leftover or leftout words.